web_widget_x2many_2d_matrix/static/src/js/2d_matrix_renderer.js

```diff
@@ -288,11 +288,24 @@ odoo.define('web_widget_x2many_2d_matrix.X2Many2dMatrixRenderer', function (requ
             }
             // Enforce mode of the parent
             options.mode = this.getParent().mode;
-            var widget = this._renderFieldWidget(
-                node, record, _.pick(options, 'mode')
-            );
-            this._handleAttributes(widget.$el, node);
-            return $td.append(widget.$el);
+            if (node.tag === 'widget') {
+                return $td.append(this._renderWidget(record, node));
+            }
+            if (node.attrs.widget || (options && options.renderWidgets)) {
+                var $el = this._renderFieldWidget(node, record, _.pick(options, 'mode'));
+                this._handleAttributes($el, node);
+                return $td.append($el);
+            }
+            var name = node.attrs.name;
+            var field = this.state.fields[name];
+            var value = record.data[name];
+            var formattedValue = field_utils.format[field.type](value, field, {
+                data: record.data,
+                escape: true,
+                isPassword: 'password' in node.attrs,
+            });
+            this._handleAttributes($td, node);
+            return $td.html(formattedValue);(widget.$el);
```

Comment:
`return $td.html(formattedValue);`**(widget.$el);** Please, remove the bold part.

Reply:
Oh, sorry, being in a rush is not helping me :)